### PR TITLE
ArcadeRatingCard

### PR DESCRIPTION
**Pull request by Bhathiya2001**
1st creat need to dev

### CODE REVIEW

File: src/components/arcadeRatingCard.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+
```
Comment (spynav) [inline src/components/arcadeRatingCard.tsx:1]: remove this space

File: src/components/arcadeRatingCard.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import { Col, Row } from 'antd';
+import { Rate } from 'antd';
+import { Button, Flex } from 'antd';
+const ArcadeRatingCard = () => {
+    return ( 
+        <Row>
+            <div style={{width:"328px", height:"191px", backgroundColor:"#EFF4FA" ,display:"flex", marginBottom:"50px", marginLeft:"25px"}} >
+
```
Comment (spynav) [inline src/components/arcadeRatingCard.tsx:9]: remove this space

File: src/components/arcadeRatingCard.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+                          <div style={{fontSize:"24px",fontWeight:"700",fontStyle:"normal", width:"150px" ,justifyContent:"center",lineHeight:"1.5"}}>SSC Complex</div>
+                          <div style={{fontSize:"11px",fontWeight:"500",fontStyle:"normal", width:"122px",justifyContent:"center",marginTop:"1px",lineHeight:"1.5",alignItems:"center",display:"flex"}}>level one cricket arcade</div>
+                        </div>                          
+                          <div style={{fill:"blue",display:"flex",justifyContent:"center",alignItems:"center",marginBottom:"2px"}}><Rate disabled defaultValue={3} style={{color:"#5587CC"}}/></div>
+                          <div style={{ width:"100%", textAlign:"center",height:"auto"}}><div>Discription/discription</div></div>
+                    </div>
+                    <div><Button type="primary" style={{backgroundColor:"#5587CC"}}>Book Arcade</Button></div>
+                </div>
+
+
```
Comment (spynav) [inline src/components/arcadeRatingCard.tsx:24]: remove these space lines

File: src/components/arcadeRatingCard.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+            </div>
+
```
Comment (spynav) [inline src/components/arcadeRatingCard.tsx:26]: remove this line

File: src/components/navbar.tsx [hunk 5]
```diff
@@ -1,10 +1,10 @@
-import { Col, Row } from 'antd';
-const Navbar = () => {
-    return ( 
-        <Row>
-            <h1>NavBar</h1>
-        </Row>
-     );
+
```
Comment (spynav) [inline src/components/navbar.tsx]: remove space

File: src/pages/home/arcadeRatingCardSection.tsx [hunk 6]
```diff
@@ -1,8 +1,32 @@
...
+                    <ArcadeRatingCard/>
+                    </Col>
+                    <Col xs={1} md={2} lg={2} xl={8}></Col>
+                    <Col xs={23} md={22} lg={22} xl={16}>
+                    <ArcadeRatingCard/>
+                    </Col>
+                </Row>
+                
+          
+
```
Comment (spynav) [inline src/pages/home/arcadeRatingCardSection.tsx:36]: remove these space lines

**Review by spynav — CHANGES_REQUESTED** [review]

File: src/components/navbar.tsx [hunk 7]
```diff
@@ -1,10 +1,10 @@
-import { Col, Row } from 'antd';
-const Navbar = () => {
-    return (
```
Comment (spynav) [inline src/components/navbar.tsx:3]: use arrow function

Comment (Bhathiya2001) [reply src/components/navbar.tsx:3]: i changed this leader
fk u

**Review by spynav — APPROVED** [review]

**Review by spynav — APPROVED** [review]

**Pull request merged**